scripts/boot_patch.sh

```diff
@@ -74,6 +75,10 @@ fi
 [ -z $PATCHVBMETAFLAG ] && PATCHVBMETAFLAG=false
 [ -z $RECOVERYMODE ] && RECOVERYMODE=false
 [ -z $SYSTEM_ROOT ] && SYSTEM_ROOT=false
+
+[ -z $DYNAMIC_PARTITIONS ] && DYNAMIC_PARTITIONS=$(grep_get_prop ro.boot.dynamic_partitions /default.prop /vendor/build.prop)
```

Comment:
pass this in magisk app like others.

Reply:
You mean via `export $DYNAMIC_PARTITIOINS` ? since that isn't done here for `$SYSTEM_ROOT` either?
Or can you point me to what you mean?

Reply:
https://github.com/topjohnwu/Magisk/blob/5e2ef1b7f41aa695d425db152add83629cc93363/app/src/main/java/com/topjohnwu/magisk/core/tasks/MagiskInstaller.kt#L492-L496

Reply:
Wouldn't boot_patch.sh then only work when called by Magisk app or there'd need to be two places to be maintained for checking dynamic partitions?

Reply:
keep consistency with other variables